code/datums/elements/weapon_description.dm

```diff
@@ -1,4 +1,4 @@
-#define HITS_TO_CRIT(damage) round(100 / damage, 0.1)
+#define HITS_TO_CRIT(damage) round(130 / damage, 0.1) //Skyrat Edit
```

Comment:
?

Reply:
uwu what's this?
Seriously though what's this for?